addon/package.json

```diff
@@ -41,47 +34,67 @@
+    "lint:js:fix": "eslint . --fix",
+    "prepack": "rollup --config",
     "start": "rollup --config --watch",
-    "test": "echo 'Please refer to test-app for addon tests'",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@embroider/addon-shim": "^1.5.0",
-    "@embroider/macros": "^1.5.0",
+    "@embroider/addon-shim": "^1.8.4",
+    "@embroider/macros": "^1.10.0",
     "assert-never": "^1.2.1",
     "ember-element-helper": "^0.5.0 || ^0.6.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.9",
-    "@babel/plugin-proposal-decorators": "^7.17.9",
-    "@babel/preset-typescript": "^7.16.7",
-    "@embroider/addon-dev": "^1.6.0",
+    "@babel/core": "^7.20.5",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-decorators": "^7.20.5",
+    "@babel/preset-typescript": "^7.18.6",
+    "@babel/runtime": "^7.20.6",
+    "@embroider/addon-dev": "^3.0.0",
     "@glimmer/env": "^0.1.7",
-    "@rollup/plugin-babel": "^5.3.1",
-    "@types/ember": "^4.0.0",
-    "@types/ember__object": "^4.0.1",
-    "@types/rsvp": "^4.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.20.0",
-    "@typescript-eslint/parser": "^5.20.0",
-    "ember-template-lint": "^4.4.2",
-    "eslint": "^8.13.0",
+    "@tsconfig/ember": "^1.0.0",
+    "@types/ember": "^4.0.3",
+    "@types/ember__application": "^4.0.4",
+    "@types/ember__array": "^4.0.3",
+    "@types/ember__component": "^4.0.0",
+    "@types/ember__controller": "^4.0.4",
+    "@types/ember__debug": "^4.0.3",
+    "@types/ember__engine": "^4.0.4",
+    "@types/ember__error": "^4.0.2",
+    "@types/ember__object": "^4.0.0",
+    "@types/ember__owner": "^4.0.2",
+    "@types/ember__polyfills": "^4.0.0",
+    "@types/ember__routing": "^4.0.0",
+    "@types/ember__runloop": "^4.0.0",
+    "@types/ember__service": "^4.0.0",
+    "@types/ember__string": "^3.16.0",
+    "@types/ember__template": "^4.0.0",
+    "@types/ember__test": "^4.0.0",
+    "@types/ember__test-helpers": "^2.6.1",
+    "@types/ember__utils": "^4.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.46.0",
+    "@typescript-eslint/parser": "^5.46.0",
+    "concurrently": "^7.6.0",
+    "ember-cli-htmlbars": "^6.1.1",
+    "ember-source": "*",
+    "ember-template-lint": "^5.2.0",
+    "eslint": "^8.28.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-ember": "^10.6.0",
+    "eslint-plugin-ember": "^11.2.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.6.2",
-    "rollup": "^2.70.2",
-    "rollup-plugin-ts": "^2.0.7",
-    "tslib": "^2.4.0",
-    "typescript": "^4.6.3"
+    "eslint-plugin-prettier": "^4.2.1",
+    "prettier": "^2.8.1",
+    "rollup": "^3.7.3",
+    "rollup-plugin-ts": "^3.0.2",
+    "typescript": "^4.9.4",
+    "webpack": "^5.75.0"
   },
   "peerDependencies": {
     "@ember/test-helpers": "^2.6.0"
```

Comment:
Hi @SergeAstapov,

Is it also ok to update the `peerDependencies` so there are no conflicts when updating `@ember/test-helpers` to its current version? (Which is now `3.2.0`)

Reply:
@Shishouille yes! I should get back to this shortly and add `@ember/test-helpers` v3 to `peerDependencies` and then have a new release